v0/destinations/facebook_pixel/transform.js

```diff
@@ -588,6 +588,17 @@ const processEvent = (message, destination) => {
           })
           .build();
       }
+      if (typeof message.event !== "string") {
```

Comment:
We should do this validation very early in the execution.

Reply:
It is inside the scope of the switch after finding type of the event.

Reply:
Needs some refactoring but it's fine for now.